--- cygwinccompiler.py	2021-12-28 03:22:24.067734554 +0900
+++ cygwinccompiler_after.py	2021-12-28 03:22:39.893232658 +0900
@@ -90,7 +90,7 @@
             return ['msvcr120']
         elif 1900 <= int(msc_ver) < 2000:
             # VS2015 / MSVC 14.0
-           return ['ucrt', 'vcruntime140'] 
+           return ['vcruntime140'] 
         else:
             raise ValueError("Unknown MS Compiler version %s " % msc_ver)
 
